bin/cask

```diff
@@ -323,6 +323,21 @@ def exec_command(command):
         raise ExecCommandError(command, error)
 
 
+def exec_emacs(command):
+    """Execute Emacs with the proper Cask environment.
+
+    ``command`` is a list of strings, containing the arguments to pass to the
+    emacs process. The Emacs executable is choosen according to the normal
+    rules (see ``get_cask_emacs``). Set ``$PATH`` and ``$EMACSLOADPATH`` to
+    include the Cask package environment.
+
+    This function replaces the current process.  It does **not** return.
+    """
+
+    command.insert(0, get_cask_emacs())
```

Comment:
Please don't modify lists.  You should consider them read-only under all circumstances, all the more if they are not local.

Use `exec_command([get_cask_emacs()] + command)` for the body of this function instead.